sdcm/sct_runner.py

```diff
@@ -381,15 +381,16 @@ def create_instance(self, test_id: str, test_duration: int,  # pylint: disable=t
                          self.region_name, self.CLOUD_PROVIDER, self.region_name)
             return None
 
-        tags = {
+        tags = TestConfig.common_tags()
+        tags.update({
             "TestId": test_id,
+            "TestName": test_name,
             "NodeType": self.NODE_TYPE,
-            "RunByUser": get_username(),
```

Comment:
why are you removing this?

Reply:
Because it's already exists in TestConfig.common_tags

Reply:
correct. But at this point, its value is None

Reply:
this is the kind of information that should be in the commit message + commit description

Reply:
@fgelcer - done